src/nvim/ex_getln.c

```diff
@@ -166,7 +166,7 @@ typedef struct command_line_state {
   int firstc;
   long count;
   int indent;
-  int c;
+  bool c;
```

Comment:
This should remain an int, from what I saw in `command_line_execute` for example.

Reply:
You are completely right, I had a brain fart.